### ISSUE TITLE
Budget info

### ISSUE THREAD

**Issue by dotexx** [Bertrand256/dash-masternode-tool#15]
It's look like budget info is incorrect.
In app only 46% are allocated, but dashninja shows 105%

Next superblock date: 8/2/2018 1:58 AM
Budget approved: 2848 Dash (46%)
Budget available: 6177 Dash
Voting deadline: 7/30/2018 4:43 AM (19 days)
Budget requested: 9308 Dash (151%)

https://www.dashninja.pl/governance.html#budgetinfo

Next super-block height: 913880
Expected on: 8/3/2018, 12:47:12 AM
Time left: 3 weeks 2 days 17 hours 17 minutes 49 seconds
Providing: 6,176.72 DASH
Unallocated: -288.25 DASH
Allocated (105%)

**Comment by dotexx** [comment]
OK. Could you add how much money we have left, if all will go thought?
Basically, dashninja shows that we already approved enough proposals  or about do so. DMT shows only we already committed to spend`.

**Comment by Bertrand256** [comment]
Yes, I have implemented it already - will be published in the next version.
